src/Platform_Core/org/lobobrowser/security/PermissionSystem.java

```diff
@@ -86,6 +86,13 @@ public void dump() {
     boards.get(0).dump();
     getLastBoard().dump();
   }
+  
+  public String dumpPermissions() {
+	  StringBuilder permissions = new StringBuilder();
+	  permissions.append(boards.get(0).dumpString());
+	  permissions.append(getLastBoard().dumpString());
```

Comment:
* Please make it iterate over all boards, not just first and last one.
* Also, I think `getPermissionsAsString()` would be a better function name. I usually use `dump` for debugging to stdout.